packages/aws-cdk/lib/api/cxapp/stacks.ts

```diff
@@ -142,52 +147,53 @@ export class AppStacks {
    *
    * Renames are *NOT* applied in list mode.
    */
-  public async listStacks(): Promise<cxapi.SynthesizedStack[]> {
+  public async listStacks(): Promise<cxapi.ICloudFormationStackArtifact[]> {
     const response = await this.synthesizeStacks();
-    return topologicalSort(response.stacks, s => s.name, s => s.dependsOn || []);
```

Comment:
Guaranteed to be already sorted?

Reply:
yes, `assembly.stacks` are topologically sorted.